setup.py

```diff
@@ -69,7 +69,7 @@ def build_extensions(self):
             # Config and build the extension
             cmd = ['cmake', ext.cmake_lists_dir] + cmake_args
             subprocess.check_call(cmd, cwd=tmpdir)
-            cmd = ['cmake', '--build', '.', '--config', BUILD_TYPE]
+            cmd = ['cmake', '--build', '.','--parallel','8', '--config', BUILD_TYPE]
```

Comment:
This is a bit too "magic-numbery" — for lack of a better term — for my taste. It's not really relevant to updating the upstream dependencies.

Reply:
Totally agree with you, I made that for my tests but I should not have pushed it. I will remove it.